module/VuFind/src/VuFind/Search/Favorites/Results.php

```diff
@@ -262,7 +262,7 @@ public function getListObject()
             $filters = $this->getParams()->getRawFilters();
             $listId = $filters['lists'][0] ?? null;
             $this->list = (null === $listId)
-                ? null : $this->listTable->getExisting($listId);
+                ? null : $this->listService->getEntityById(\VuFind\Db\Entity\UserList::class, $listId);
```

Comment:
Is there a reason you made a direct getEntityById call here instead of using the new getExisting method?

Reply:
This was done before the migration of getExisting method and missed to update it later.